### ISSUE TITLE
Cannot serialize package descriptor with tabular data profile

### ISSUE THREAD

**Issue by ronnix** [frictionlessdata/frictionless-py#1261]
We’re using Frictionless framework v5 beta.

We’d like to serialize a package descriptor with an explicit `tabular-data-package` profile.

Appartently, it’s not possible to serialize a resource descriptor with a `data-resource` or `tabular-data-resource` profile, or a package descriptor with the `data-package` or `tabular-data-package` profile.

The serialization fails with: `frictionless.exception.FrictionlessException: [metadata-error] Metadata error: cannot retrieve metadata "tabular-data-resource" because "[Errno 2] No such file or directory: 'tabular-data-resource'"`

The issue happens in `metadata_validate`, where the code expects the profile to be the path to a schema.

Should builtin profiles like  `data-resource` or `tabular-data-resource` be treated in a specific way?

Here are some tests to demonstrate the issue:
```python
@pytest.mark.parametrize("profile", ["data-resource", "tabular-data-resource"])
def test_data_resource_profile(profile):
    resource = Resource(
        name="table", path="data/table.csv", profiles=[profile]
    )
    descriptor = resource.to_descriptor()
    assert descriptor["profile"] == profile


@pytest.mark.parametrize("profile", ["data-package", "tabular-data-package"])
def test_data_package_profile(profile):
    resource = Resource(name="table", path="data/table.csv")
    package = Package(resources=[resource], profiles=[profile])
    descriptor = package.to_descriptor()
    assert descriptor["profile"] == profile
```

**Comment by fjuniorr** [comment]
@shashigharti and @roll I noticed that in 5.7.1 a similar error is still being raised when running

```bash
frictionless describe --standards v1 --type package https://raw.githubusercontent.com/transparencia-mg/datapackage-reprex/root-datapackage/estados.csv
# FrictionlessException: [metadata-error] Metadata error: cannot retrieve metadata "tabular-data-profile" because "[Errno 2] No such file or directory: 'tabular-data-profile'" 
```

Also, not sure if I should open a new issue for this but the `profile` property is being removed in both package and resources. Here's a reprex

```python
from frictionless import Package

package = Package.from_descriptor('https://raw.githubusercontent.com/transparencia-mg/datapackage-reprex/root-datapackage/datapackage.json')

package.profile # None
package.get_resource('estados').profile # None
```

This is currently a practical problem for me[^20230227T173423]  because `frictionless-r` does not support reading a resource without a `tabular-data-resource` profile, and therefore I can't use `frictionless-py` to generate data packages that work on both clients.

After #1275 is this the intended behaviour ?

[^20230227T173423]:  Without discussing where the spec future directions, such as in https://github.com/frictionlessdata/frictionlessdata.io/discussions/746

**Comment by shashigharti** [comment]
Thanks @fjuniorr for the question. I think it is being fixed by @roll in this new [commit](https://github.com/frictionlessdata/framework/pull/1437/commits/d4c1bcb2e5f8fbc7fef7355f8b14d05c36c6c5d2):
https://github.com/frictionlessdata/framework/blob/main/frictionless/package/package.py#L83

**Comment by roll** [comment]
Hi @fjuniorr,

Thanks a lot for reporting!

It's been fixed in `frictionless@5.7.2`

**Comment by fjuniorr** [comment]
Thanks for the quickfix! 

The CLI `frictionless describe --standards v1` is indeed working, however the `profile` is still being stripped when importing/exporting a descriptor:

```python
from frictionless import Package

url = 'https://raw.githubusercontent.com/transparencia-mg/datapackage-reprex/root-datapackage/datapackage.json'
package = Package.from_descriptor(url)
package.to_json('datapackage.json') # profile properties will not be here
```

Is this a separate issue?

**Comment by roll** [comment]
Hi @fjuniorr,

In Python it works similarly:

```python
from frictionless import Package, system
system.standards = "v1"

package = Package.from_descriptor(
    "https://raw.githubusercontent.com/transparencia-mg/datapackage-reprex/root-datapackage/datapackage.json"
)
print(package)
```

or 

```python
from frictionless import Package, system

package = Package.from_descriptor(
    "https://raw.githubusercontent.com/transparencia-mg/datapackage-reprex/root-datapackage/datapackage.json"
)

with system.use_context(standards="v1"):
    print(package)
```

**Comment by roll** [comment]
BTW @fjuniorr 
Does `frictionless-r` support providing a profile as a path/url? It's also supported by v5 in Python

**Comment by fjuniorr** [comment]
> Does `frictionless-r` support providing a profile as a path/url? It's also supported by v5 in Python

It allows reading, but I don't think it does actual metadata validation.

**Comment by roll** [comment]
That's the problem with v1's profile is trying to be 2 things at the same time: indicating the type of the resource and providing additional validation rules. In-general these 2 things are unrelated so one should be able to have a `TableResoruce` with e.g. fiscal profile (fiscal specific metadata)

**Comment by fjuniorr** [comment]
Yeah, I can see clearly those two concerns for the `profile` at the data package level and some of the practical problems on this issue https://github.com/frictionlessdata/specs/issues/743
